cmd/otel-allocator/README.md

```diff
@@ -4,6 +4,135 @@ The TargetAllocator is an optional separately deployed component of an OpenTelem
 distribute targets of the PrometheusReceiver on all deployed Collector instances. The release version matches the
 operator's most recent release as well.
 
+In essence, Prometheus Receiver configs are "stolen" and replaced with a http_sd_configs directive that points to the 
+Allocator, these are then loadbalanced/sharded to the collectors. In addition, the TargetAllocator can discover targets via Prometheus CRs (currently ServiceMonitor,
+PodMonitor) which it presents on the `/jobs` endpoint.
```

Comment:
The configuration is provided by the `/scrape_configs` endpoint, and the resultant jobs are on the `/jobs` endpoint

Reply:
Fixed